### DISCUSSION TITLE
No logro guardar en formato ".py"

### DISCUSSION THREAD

**Discussion by MarioBerne** [LaunchX-InnovaccionVirtual/CursoIntroPython#315] [Q&A]
La instrucción es guardar en formato ".py" sin embargo me aparece error. Reinicio Visual Studio Code y ya no me aparece error pero simplemente no lo guarda. Hay problema si lo guardo y envio las katas en extensión "IPYNB"?

**Comment by JavierMendez-Coder** [comment]
Buenas noches. 
Si puedes brindar más información como:
1. Módulo de la kata.
2. Imagen del error.
4. Datos que consideres relevantes.

Necesito más contexto. Gracias 👋.

**Reply by OscarIslasH** [reply]
Es la kata 10

**Reply by JavierMendez-Coder** [reply]
Ahh, en ese caso no creo que haya problema alguno siempre y cuando puedan replicar algunos de los errores que abordan en los ejercicios.
De todas formas, creo que el error que sucede se debe a que se está intentado guardar un archivo con extensión _.ipynb_ como _.py_. Entonces, a la hora de ejecutarlo el compilador de python este no comprende lo que esté leyendo, ya que, cada uno poseen sintaxis diferentes.

---
### En caso de que desees subirlos como _.py_:
1. Puedes crear un nuevo documento de texto.

![image](https://user-images.githubusercontent.com/99059210/154814774-f5a75d3c-7316-4ee5-9334-1140292399be.png)

2. Agregar el código de la asignación.

![image](https://user-images.githubusercontent.com/99059210/154814869-31dde589-49cb-4d9b-833f-bd79d13c8470.png)

3. Por último, guardarlo con el nombre indicado en el ejercicio (si te lo brindan), y cambiar la extensión del documento.

![image](https://user-images.githubusercontent.com/99059210/154815050-76083fa2-b82e-4b8a-ae08-44450c273b25.png)

Algo importante por destacar es que en mi caso yo tengo las extensiones de los documentos visibles para asegurarme de este se encuentre guardado con la extensión correcta. Esta opción se puede activar facilmente en tu explorador de archivos dirigiendote a la pestaña _vista_, y seleccionando la casilla _nombre de la extensión del archivo_.

![image](https://user-images.githubusercontent.com/99059210/154815229-380015d1-7e0b-4fa5-8943-041ac52087fa.png)

**Comment by Adagiominor** [comment]
Si instalaste la ultima version de python:

en terminal:

cd + 'direccion especifica de la carpeta donde guardaras tu py '

python > app.py

y asi creas cualquier archivo .py, despues solo copia y pega varias vece el mismo archivo vacio para no tener que ejecutar cada n veces que quieras otro archivo

**Comment by JuanSalvador73** [comment]
´¿Bajaste el jupiter para VS?
